strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/JavaElement.java

```diff
@@ -0,0 +1,63 @@
+package org.arquillian.smart.testing.strategies.affected;
+
+import java.io.File;
+import org.arquillian.smart.testing.strategies.affected.ast.JavaClass;
+
+public class JavaElement {
+
+    private final String className;
+    private final File classFile;
+
+
+    public JavaElement(String className) {
+        this.className = className;
+        this.classFile = null;
+    }
+
+    /**
+     * Constructor for creating a Java element.
+     * @param className of the java element.
+     * @param classFile location of the java element. Can be null.
```

Comment:
Now nothing since this is only used when `JavaElement` is a test which is known because it is the leaf of the graph. This can be null in case of classes that are inside a JAR.